client/packages/purchase_orders/src/DetailView/Footer/utils.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  LocaleKey,
+  PurchaseOrderNodeStatus,
+  TypedTFunction,
+} from '@openmsupply-client/common';
+import { PurchaseOrderLineFragment } from '../../api';
+
+export const purchaseOrderStatuses: PurchaseOrderNodeStatus[] = [
+  PurchaseOrderNodeStatus.New,
+  PurchaseOrderNodeStatus.Authorised,
+  PurchaseOrderNodeStatus.Confirmed,
+  PurchaseOrderNodeStatus.Finalised,
+];
+
+export const statusTranslation: Record<PurchaseOrderNodeStatus, LocaleKey> = {
+  NEW: 'label.new',
+  AUTHORISED: 'label.authorised',
+  CONFIRMED: 'label.confirmed',
+  FINALISED: 'label.finalised',
+};
+
+export const validateEmptyPurchaseOrder = (
+  lines:
+    | {
+        totalCount: number;
+        nodes: PurchaseOrderLineFragment[];
+      }
+    | undefined
+): boolean => {
+  if (!lines) return false;
+
+  const noRequestedPackSize = lines.nodes.every(
+    line => line.requestedPackSize === 0
+  );
+  if (lines.totalCount === 0 || noRequestedPackSize) return false;
+  return true;
+};
+
+export const getStatusTranslator =
+  (t: TypedTFunction<LocaleKey>) =>
+  (status: PurchaseOrderNodeStatus): string =>
+    t(
+      statusTranslation[status] ??
```

Comment:
I don't really know if this is an actual cause for concern, but it would make more sense to me to fall back to something louder like 'unknown status' if the statusTranslation dictionary is no longer comprehensive after some status enum update.

But probably fine?

Reply:
Thats a good point and would make more sense. I would update to be so but maybe can be done as a refactor issue? Saying that because invoices seems to also fallback to new. That being said, should be fine for now imo